docs/sources/flow/reference/components/prometheus.operator.servicemonitors.md

```diff
@@ -192,3 +198,26 @@ prometheus.operator.servicemonitors "services" {
     }
 }
 ```
+
+This example will apply additional relabel rules to discovered targets for hashmod sharding. Here we have five shards, and an environment variable provides our current agent's shard.
```

Comment:
Can we say that we expect to have 5 agents running with this set in the 0-4 range? eg. People may not be aware and try 1-5